packages/libs/map/routing.ts

```diff
@@ -108,21 +124,18 @@ export function findRoute(
       }
     }
   }
-
-  console.warn(
-    `no ${direction} route found from`,
-    start,
-    'and',
-    goal,
-    `(${numIters} iterations)`,
-  );
-  return undefined;
+  return {
+    success: false,
+    key,
+    mode,
+    numIters,
```

Comment:
The value of `numIters` is usually a good indicator of _where_ routing failed. If the value is high (like, 250k), then the graph's problem is likely near the destination (aka `end`) node. But if it's low (like, less than 100), the problem is likely near the origin (aka `start`) node.

As an example, one unrouteable origin/destination company pair currently being flagged is

```json
{
  "numIters": 258196,
  "start": {
    "company": "ali_car_dlr.hays",
    "nodeUid": "541ac1020c853cdb",
    "latLng": "/38.984/-99.311"
  },
  "end": {
    "company": "syn_che_plnt.st_joseph",
    "nodeUid": "541ac10150054bfe",
    "latLng": "/39.703/-95.177"
  }
}
```

The high `numIters` value suggests the problem lies with some company in St Joseph... which is in Missouri, which is still unreleased, and isn't surprising that no route to it could be found (especially since there are no roads connecting it to the rest of the road network!)